corehq/apps/cloudcare/static/cloudcare/js/debugger/debugger.js

```diff
@@ -152,6 +152,7 @@ hqDefine('cloudcare/js/debugger/debugger', function () {
             }
         ).done(function(response) {
             this.formattedQuestionsHtml(response.formattedQuestions);
+            hqImport("reports/js/readable_form").init();
```

Comment:
(same question about init())

Reply:
Yeah, similarly, this questions don't exist on page load; cloudcare does a separate request for `formatted_questions` to get this content.